web-share/src/svelte/components/Media.svelte

```diff
@@ -0,0 +1,94 @@
+<!--
+Copyright 2019 Google, Inc.
+
+Licensed under the [Apache License, Version 2.0](LICENSE) (the "License");
+you may not use this file except in compliance with the License. You may
+obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<script>
+	import Button from './Button.svelte';
+
+	export let contentType = '';
+	export let src = '';
+
+	const canShare = 'canShare' in navigator;
+
+	function handleView() {
+		window.location.href = src;
+	}
+
+	async function handleDelete() {
+		const cache = await caches.open('media');
+		await cache.delete(src);
+		window.location.reload();
+	}
+
+	async function handleShare() {
+		const cache = await caches.open('media');
```

Comment:
Should this be `cacheName` as well here?

Reply:
Yup!